src/components/BackgroundLayout/index.js

```diff
@@ -2,16 +2,14 @@ import React, { useEffect, useState } from 'react';
 
 import defaultImage from 'images/auth_background.png';
 
-const BackgroundLayout = ({ dynamicImage, noBackground, children }) => {
+const BackgroundLayout = ({ dynamicImage, children }) => {
   const [backgroundImage, setBackgroundImage] = useState(defaultImage);
 
   useEffect(() => {
-    if (noBackground) {
-      setBackgroundImage(null);
-    } else if (dynamicImage) {
+    if (dynamicImage !== undefined) {
```

Comment:
I think `if (dynamicImage)` is enough, it covers the case when `dynamicImage` is empty string as well.

Reply:
The thing is, there are three types of background, One with the [default image](https://github.com/rafayet-monon/surveyor/blob/23898e241599d9c0aa71c074d9e36bc9c0dc66fd/src/components/BackgroundLayout/index.js#L3), one with [dynamic images](https://drive.google.com/file/d/1iFqi4Exi4krF8g4YoU3BGUxgsty02QfW/view) which will be changed with respect to survey image. And there's one more which will have [no images in background](https://drive.google.com/file/d/1C2brmVbM3oebpDX6-c4Wa3FF5l78ADib/view). In the last scenario, the background will fallback to  [background-color](https://github.com/rafayet-monon/surveyor/blob/fc0f49a1fc23463bb05aace7b52c492cd2401abd/src/stylesheets/components/_background-layout.scss#L10).

For these reasons, when there's [no prop provided](https://github.com/rafayet-monon/surveyor/blob/chore/task-10-update-code-conventions/src/screens/Login/page.js#L12) it will have `undefined` and will get `defaultImage` and if [null is provided](https://github.com/rafayet-monon/surveyor/blob/chore/task-10-update-code-conventions/src/screens/LazyLoader/page.js#L8) it will fallback to `background-color`.

@namtx Do you think this is too complicated? How can I make it better?

Reply:
It makes sense. Yup, we can go with the current solution.